zigate/protocol/response.py

```diff
@@ -144,6 +144,21 @@ def __str__(self):
 
 
 
+@register(0x8102)
+class IndividualAttributeReport(Response):
+
+    def __init__(self, *args):
+        super().__init__(*args)
+
+        self.struct = struct.Struct('!BHBHHBBH')
```

Comment:
The struct isn't used elsewhere so it doesn't have to be an attribute of the object.